package.json

```diff
@@ -21,6 +21,8 @@
   },
   "homepage": "https://github.com/Synthetixio/erc7412#readme",
   "dependencies": {
+    "@pythnetwork/pyth-evm-js": "^1.25.0",
```

Comment:
maybe this should be a peer dependency or something? not everyone who uses the `erc7412` lib may use pyth, so inclusion of this library would be excess on frontends